Header header
sensor_msgs/Image rgb_wrist
sensor_msgs/Image depth_wrist
sensor_msgs/Image rgb_static
sensor_msgs/Image depth_static
float64[] joint_positions
geometry_msgs/Pose end_effector_pose